README.md

```diff
@@ -47,19 +60,42 @@ val v1CartCreatedPersister = persister[CartCreatedV1]("cart-created")
 
 // spray-json persister for V2 but with support for migration
 // of data writen in the V1 format.
-val v2CartCreatedPersister = persister[CartCreatedV2, V2]("cart-created",
-  from[V1].to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
+val v2CartCreatedPersister = persister[CartCreatedV2, V2](
+  "cart-created",
+  from[V1]
+    .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
 )
 
 // spray-json persister for V3 but with support for migration
 // of data writen in the V1 and V2 formats.
-val v3CartCreatedPersister = persister[CartCreatedV3, V3]("cart-created",
+val v3CartCreatedPersister = persister[CartCreatedV3, V3](
+  "cart-created",
   from[V1]
     .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
     .to[V3](_.update('timestamp ! set[Long](System.currentTimeMillis)))
 )
 ```
 
+To use persisters, they need to be encapsulated in a `StaminaAkkaSerializer` that is registered, in the 
+application.conf, for the event classes that need to be persisted. Here the provided `Persistable` marker trait is used
+and the serializer is registered for `Persistable` classes.
+
+```scala
+class CartCreatedV3(...) extends Persistable
```

Comment:
On re-reading this I would make this a separate code block and perhaps add a separate paragraph about the marker trait stuff to it since putting it so close to the serializer makes no sense and would never happen in a real application.